language-plutus-core/test/Evaluation/Golden/verifySignature.plc.golden

```diff
@@ -0,0 +1 @@
+(error (all a (type) (fun a (fun a a))))
```

Comment:
@vmchale, @michaelpj, this can be an error in my code, but I suspect that there is something wrong about the way we handle `verifySignature`. In any case we need a test on the current master that succeeds (i.e. returns the encoding of `true`).

Reply:
I think this is a regression from `master`. I'll push the correct file.